themes/delphi/assets/css/layouts.scss

```diff
@@ -0,0 +1,36 @@
+.content-grid {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    grid-gap: $grid-gap;
+}
+
+.grid-2-8 {
+    grid-column: 2 / span 6;
+}
+.grid-8-11 {
+    grid-column: 8 / 12;
+}
+.grid-3-11 {
+    grid-column: 3 / 11;
+}
+.grid-2-12 {
+    grid-column: 2 / 12;
+}
+.grid-1-2 {
+    grid-column: 2 / 12;

```

Comment:
why is this the same as 2-12?

Reply:
This was unintentional, please take a look again.